README.md

```diff
@@ -153,6 +155,18 @@ COMMAND SUPPLIED
     git-xargs --version
     ```
 
+### Installion option 2: Run go get 
+
+1. **Run go get to install the latest release of git-xargs**
+
+`go get github.com/gruntwork-io/git-xargs/cmd/git-xargs`
+
+1. **Alternatively, use go get to install a specific release of git-xargs**
+
+`go get github.com/gruntwork-io/git-xargs/cmd/git-xargs@v0.0.4`
```

Comment:
I think this should be quickly updated to `v0.0.5` since `v0.0.4` was just released, and doesn't contain this fix.

Reply:
Done in e9c1276